Samples/README.md

```diff
@@ -63,6 +63,16 @@ namespace UwpSample
 }
 ```
 
+## CreateHubSample
+Sample dotnet core WebApi application for sending push notifications.
+
+To run the sample update the `/WebApiSample/WebApiSample/Models/Notification.cs` file and execute the sample as such:
```

Comment:
Be a little bit more explicit here such as "Update the `/WebApiSample/WebApiSample/Models/Notification.cs` with your Notification Hubs `DefaultFullSharedAccessSignature` Access Policy connection string.